misc/v3-task-endpoint-validator-windows/v3-task-endpoint-validator-windows.go

```diff
@@ -0,0 +1,387 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
What's the difference of the file between windows and linux? Also is it possible to reuse the taskmetadata validator file?

Reply:
The difference between windows and linux: we need to verify three network modes for linux, but we only need to verify one network mode for Windows, also windows has a different task name. 
This means the verification logic and some of the metadata fields are different. Also we may need to do some extra configuration to reuse the file in another folder.

is it possible to reuse the task metadata validator file: theoretically it's possible, but it will result in a very confusing configuration and verification logic.

In sum, I don't think it's worth to put them together.